book/src/dev/rfcs/0010-v5-transaction.md

```diff
@@ -140,16 +142,13 @@ V5 {
     expiry_height: block::Height,
     inputs: Vec<transparent::Input>,
     outputs: Vec<transparent::Output>,
-    shielded_data: Option<SaplingV5ShieldedData>,
-    value_balance_sapling: Amount,
-    output_proofs_sapling: Vec<Groth16Proof>
-    anchor_sapling: tree::Root,
-    spend_zkproof_sapling: Vec<Groth16Proof>,
-    spend_auth_sigs_sapling: Vec<redjubjub::Signature<SpendAuth>>
-    orchard_data: Option<OrchardData>,
-    flags_orchard: u8,
-    value_balance_orchard: Amount,
-    anchor_orchard: tree::Root,
+    sapling_value_balance: Amount,
+    sapling_anchor: tree::Root,
+    sapling_shielded_data: Option<SaplingV5ShieldedData>,
+    orchard_flags: OrchardFlags,
+    orchard_value_balance: Amount,
+    orchard_anchor: tree::Root,
+    orchard_shielded_data: Option<OrchardShieldedData>,
```

Comment:
@dconnolly should we move `sapling_anchor` and `orchard_anchor` into `ShieldedData`?

Reply:
Also what about `orchard_flags`?

Reply:
GitHub has hidden these questions as "Outdated", but they haven't been dealt with, so I'm going to re-post them.